vllm/v1/sample/logits_processor/__init__.py

```diff
@@ -185,6 +191,17 @@ def build_logitsprocs(
             " do not support logits processors."
         )
         return LogitsProcessors()
+
+    # Check if speculative decoding is enabled.
+    if vllm_config.speculative_config:
+        if custom_logitsprocs:
+            raise ValueError(STR_SPEC_DEC_REJECTS_LOGITSPROCS)
+        logger.debug(
+            "Skipping logits processor loading because "
+            "they are not supported when speculative decoding is enabled."
```

Comment:
actually given the above we may want to change this to a warning and change the message
```suggestion
        logger.warning(
            "min_p, logit_bias, and min_tokens parameters won't currently work "
            "with speculative decoding enabled."
```